pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 
 dependencies:
   async: '>1.9.0 <3.0.0'
-  protobuf: '>=0.10.4 <0.12.0'
+  protobuf: '>=0.10.4 <0.13.0'
```

Comment:
Actually wait, this should be <0.14.0 :).

Reply:
`0.13.0` was just released today. 😄